index.md

```diff
@@ -4,78 +4,47 @@ title: "FOSDEM &lt;3 Ruby"
-* Costs and Opportunities provided by the multitude of implementations
-
-
-### Involved projects
-
-* [JRuby](http://jruby.org)
-* [Ruby (MRI)](https://www.ruby-lang.org/)
-* [RubyTruffle](http://www.chrisseaton.com/rubytruffle/)
-* [Rubinius (Ruby on Ruby)](http://rubini.us/)
-
-
 ## Why we're here
 
-While many Ruby developers attend FOSDEM, the Ruby open source community has
-generally been underrepresented in the technical content of the event. With the
-Ruby devroom not only would the Ruby community become more plugged into the
-FOSDEM experience, but FOSDEM would be hosting Ruby-oriented content that is
-both unique and very technical. Considering the broad interest in dynamic and
-interpreted languages, the Ruby devroom will also provide compelling technical
-content interesting to members of the Python, Perl, JavaScript, Smalltalk, LLVM
-and Java communities. There is quite some interest in the topic, as the
-overcrowded Java dev room if  [Charles Oliver
-Nutter's](https://github.com/headius) talk on JRuby in the FOSDEM 2014 Free
-Java devroom is any indication.
+Join us at the Ruby Devroom at FOSDEM 2023 for a celebration of all things Ruby! 
+We invite all Ruby enthusiasts to share their knowledge and passion through presentations, workshops, and discussions.
 
-Considering the number of notable Ruby implementations, listed below, there
-should be more than enough highly interesting and technical content!
+This devroom will provide a platform for Ruby developers to share their experiences, learn from each other, and collaborate on projects. 
+We welcome submissions from developers of all levels, from seasoned veterans to those just starting out. 
+If you have a story to share or a lesson learned, we want to hear from you
 
-* Ruby (MRI, the original C-based VM)
-* JRuby (Ruby on the JVM)
-* Rubinius (Ruby on Ruby)
-* Topaz (Ruby on RPython, the toolkit that PyPy built)
-* RubyTruffle (JRuby using Oracle's Truffle compiler)
-* RubyMotion (Ruby on top of Apple's Objective-C/Cocoa foundation)
-* Maglev (Ruby on the GemStone Smalltalk VM)
-* Opal (A working Ruby to JavaScript transpiler)
-* mruby (lightweight, embedded Ruby)
+Topics could include:
 
-## When and Where
+- Ruby frameworks and libraries
+- Testing and debugging techniques
+- Innovative uses of Ruby in web development, automation, or beyond
+- Real-world case studies and success stories
+- Challenges and opportunities for Ruby in the future
 
-The Ruby devroom takes place on Saturday, the 4th February 2017, in the room K.4.201, [building K](https://fosdem.org/2017/schedule/buildings/#k), on the
-[Solbosch Campus, Université Libre de Bruxelles](https://goo.gl/maps/52tGkVuKJZS2).
+## When and Where
 
-## Who we are
+Important dates:
+- 1st December 2023: Submission Deadline 
+- 8th December 2023: Acceptance Notification 
+- 15th December 2023: Final Schedule Announcement
 
-{% include committee.md %}
+## Submit a talk
 
-## Code of Conduct
+Head to the FOSDEM 2024 pretalx
```

Comment:
Consider removing the new lines and use the link to CFP as `[https://pretalx.fosdem.org/fosdem-2024/cfp](https://pretalx.fosdem.org/fosdem-2024/cfp)` or something similar.

Maybe something like:
`Head to [FOSDEM 2024 Pretalx website](https://pretalx.fosdem.org/fosdem-2024/cfp)`

Reply:
```suggestion
Head to the [FOSDEM 2024 Pretalx website](https://pretalx.fosdem.org/fosdem-2024/cfp)
```